x-pack/plugins/watcher/server/models/watch_status_model/watch_status_model.test.js

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+
+import { ACTION_STATES, WATCH_STATES, WATCH_STATE_COMMENTS } from '../../../common/constants';
+import { WatchStatusModel } from './watch_status_model';
+
+describe('WatchStatusModel', () => {
```

Comment:
This diff looks significant, but all that changed was:

* Rename of `WatchStatus` -> `WatchStatusModel`
* Removal of an unnecessary root-level `describe` block